programs/zk-token-proof-tests/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "solana-zk-token-proof-program-tests"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
```

Comment:
oh, please note https://github.com/solana-labs/solana/pull/29997 where we change the boilerplate `authors` value